lib/db/dbmi_base/msvc/dirent.c

```diff
@@ -0,0 +1,113 @@
+
```

Comment:
This portability change is probably not Cmake related and could be separated out?